packages/cli/src/daemon-config.ts

```diff
@@ -77,7 +77,7 @@ export class DaemonIpfsConfig {
 }
 
 /**
- * Whether the Ceramic state store should store its data on the local filesystem or in Amazon S3.
+ * Whether the Ceramic storage should store its data on the local filesystem or in Amazon S3.
```

Comment:
IMO, that should remain as it is, if the CLI mentions the same concept as `state store`. And, IMO, it is still a state store.